# Coleta de dados para o exercício de casa:

Coletar a altura média de cada um dos alunas jogadors do time de basquete EUA nos jogos Olímpicos de Paris 2024.
Coletar a altura média dos 10 jogadores mais altos da hisória da liga NBA.


**Exercício: Teste z para Altura Média dos jogadores de basquete Norte Americanos**

**Cenário:**

Desejo avaliar se a altura média dos jogadores de basquete americanos que jogaram nos jogos Olímpicos de Paris 2024 é significativamente diferente da altura média dos 10 jogadores mais altos da NBA, que é de 228 cm.

informações:
jogadores mais altos da liga NBA >> https://olympics.com/pt/noticias/nba-jogadores-mais-altos-historia

Gheorghe Mureșan (ROM) - 2.31m
Manute Bol (SUD) - 2.31m
Tacko Fall (SEN) - 2.29m
Slavko Vraneš - 2.29m
Shawn Bradley (EUA) - 2.29m
Yao Ming (CHI) - 2.29m
Chuck Nevitt (EUA) - 2.26m
Pavel Podkolzin (RUS) - 2.26m
Sim Bhullar (CAN) - 2.26m
Victor Wembanyama (FRA) - 2.26m

Altura média dos jogadores mais altos da NBA (em cm) : [231, 231, 229, 229, 229, 229, 226, 226, 226, 226, 226]

**Objetivo:**

    Verificar se a altura média dos jogadores de basquete olímplicos estadunidenses, Paris 2024  é significativamente diferente de 228 cm.

Dados da Amostra:

    Altura dos jogadores olímpicos 2024 (em cm): [206, 191, 193, 206, 193, 198, 193, 208, 193, 213, 198, 193]

LeBron James: 2.06 m
Stephen Curry: 1.91 m
Devin Booker: 1.93 m
Kevin Durant: 2.06 m
Anthony Edwards: 1.93 m
Bam Adebayo: 1.98 m
Derrick White: 1.93 m
Anthony Davis: 2.08 m
Jrue Holiday: 1.93 m
Joel Embiid: 2.13 m
Jayson Tatum: 1.98 m
Tyrese Haliburton: 1.93 m


**Perguntas:**


Calcule a estatística z e o valor p para verificar se a altura média dos jogadores de basquete olímpicos 2024 norte americanos é significativamente diferente de 228 cm.

**Interprete os resultados:** A altura média dos jogadores de basquete olímpicos 2024 norte americanos é significativamente diferente de 228 cm?

**Hipóteses:**

**Hipótese Nula (H0):**

A hipótese nula é a suposição inicial de que não há diferença significativa entre a média da amostra e o valor de referência.
No contexto do nosso exemplo:

    H0​: A altura média dos jogadores de basquete olímpicos 2024 norte americanos é igual à altura média dos 10 jogadores mais altos da NBA.
    H0:μ = 228

**Hipótese Alternativa (H1):**

A hipótese alternativa é a afirmação que desejamos testar, que sugere que existe uma diferença significativa.

No contexto do nosso exemplo:

      H1​: A altura média dos jogadores de basquete olímpicos 2024 norte americanos é diferente da altura média dos 10 jogadores mais altos da NBA.
      H1:μ ≠ 228

### Em outras palavras...

**Hipótese Nula (H0):**

Supõe que não há diferença entre a média dos jogadores de basquete olímpicos 2024 norte americanos e a média dos 10 jogadores mais altos da NBA (228 cm). A média da altura dos jogadores de basquete olímpicos 2024 norte americanos é 198.75 cm.

**Hipótese Alternativa (H1):**

Sugere que há uma diferença entre a média dos jogadores de basquete olímpicos 2024 norte americanos e a média da dos 10 jogadores mais altos da NBA. A média da altura dos jogadores olímpicos de basquete estadunidenses, Paris 2024 não é 198.75 cm.


**Teste Bilateral:**

Neste exemplo, utilizamos um teste bilateral (ou de duas caudas), pois estamos verificando se a média média dos jogadores de basquete olímpicos 2024 norte americanos é diferente (pode ser maior ou menor) da média dos 10 jogadores mais altos da NBA, e não apenas se é maior ou menor.


In [4]:
jogadores_mais_altos_nba = [231, 231, 229, 229, 229, 229, 226, 226, 226, 226, 226]

media = sum(jogadores_mais_altos_nba) / len(jogadores_mais_altos_nba)

print("A média é:", media)


A média é: 228.0


In [5]:
import numpy as np
from statsmodels.stats.weightstats import ztest

# Dados da amostra
jogadores_olimpicos_americanos = np.array([206, 191, 193, 206, 193, 198, 193, 208, 193, 213, 198, 193])
jogadores_mais_altos_nba = 228

# Teste z para uma amostra
z_statistic, p_value = ztest(jogadores_olimpicos_americanos, value= jogadores_mais_altos_nba)
print("Teste z para uma amostra:")
print(f"z-statistic: {z_statistic:.4f}, p-value: {p_value:.4f}")

Teste z para uma amostra:
z-statistic: -13.4937, p-value: 0.0000


**Respostas Esperadas:**

Teste z para uma amostra:


    * Se o valor p (0.000) for menor que 0.05, a altura dos jogadores de basquete olímpico americanos é significativamente diferente de 228 cm.

    A interpretação do valor do teste z para uma amostra, com uma estatística z de -13.4937 e um valor p de 0.0000, é a seguinte:

**Interpretação:**

**Estatística z (z-statistic): -13.4937**

  A estatística z indica quantos desvios padrão a média da amostra está distante da média de referência (228 cm) sob a hipótese nula.
  Um valor negativo da estatística z sugere que a média da amostra está acima da abaixo de referência, mas precisamos do valor p para determinar a significância.

  Por exemplo, se a estatística z é -13, isso indica que a observação está 13 desvios-padrão abaixo da média. Quanto mais negativo o valor de z, mais distante (e abaixo) da média está a observação.

**Valor p (p-value): 0.0000**

  O valor p representa a probabilidade de observar um valor da estatística z tão extremo ou mais extremo do que o valor observado, assumindo que a hipótese nula é verdadeira.
  Neste caso, um valor p de 0.0000 significa que há aproximadamente 0.0% de chance de obter a média da amostra observada (ou uma mais extrema) sendo a média dos jogadores mais altos da NBA 228 cm.

**Conclusão:**

Nível de Significância: Um nível de significância comum usado em testes estatísticos é 0.05 (ou 5%). Se o valor p for menor que 0.05, rejeitamos a hipótese nula.
Neste caso: O valor p (0.000) é menor que 0.05. Portanto, rejeitamos a hipótese nula.

**Interpretação Final:**

A altura média dos jogadores de basquete olímpico da seleção norte americana é significativamente diferente de 228 cm. Em outras palavras, há evidências estatisticamente significativas para afirmar que a altura média dos jogadores estadunidendes de basquete nas Olímpiadas de Paris 2024 difere da altura média dos 10 jogadores mais alto da NBA de 228 cm.